src/wwwroot/partials/signup/confirmationPage.html

```diff
@@ -18,24 +18,24 @@
       <div class="flex margin--bottom-inputs">
         <div class="wrapper-input width--full margin--right-inputs">
           <label class="label--regular" for="industry">{{'industry_text' | translate}}:</label>
-          <ui-select ng-model="selected.value" name="industry" title="{{'select_choose_industry' | translate}}" on-select="form.industry.$modelValue = $model" required>
+          <ui-select id="confirmation-select-industry"  ng-model="selected.value" name="industry" title="{{'select_choose_industry' | translate}}" on-select="form.industry.$modelValue = $model" required>
```

Comment:
Personally, I prefer a simpler id, like `field_industry`, I do not imagine a collision situation now, and when it happen we could analyze the situation and find the better solution.

If you insist on defining a convention, I prefer something simpler like `[controllerName]-[fieldName]` or better yet `[formName]-[fieldName]`.

**BTW You should update the _for_ value from the related label.**